feature/fortune/src/main/java/org/sopt/official/feature/fortune/feature/fortuneDetail/component/PokeRecommendationUserProfileImage.kt

```diff
@@ -0,0 +1,46 @@
+package org.sopt.official.feature.fortune.feature.fortuneDetail.component
+
+import androidx.compose.foundation.Image
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import org.sopt.official.designsystem.SoptTheme
+import org.sopt.official.feature.fortune.R.drawable.ic_empty_profile
+import org.sopt.official.feature.fortune.component.UrlImage
+
+@Composable
+internal fun PokeRecommendationUserProfileImage(
+    profile: String,
+    isEmptyProfile: Boolean,
+    modifier: Modifier = Modifier,
+) {
+    when (isEmptyProfile) {
+        true -> {
+            Image(
+                painter = painterResource(ic_empty_profile),
+                contentDescription = "profileImageEmpty",
+                modifier = modifier,
+            )
+        }
+
+        false -> {
+            UrlImage(
+                url = profile,
+                contentDescription = "profileImage",
+                modifier = modifier,
+            )
+        }
+    }
+}
```

Comment:
음? 이거 이렇게 구현한 이유가 있을까?

차라리 `profile: String = ""`

```suggestion
    when (profile.isBlankOrEmpty()) {
        true -> {
            Image(
                painter = painterResource(ic_empty_profile),
                contentDescription = "profileImageEmpty",
                modifier = modifier,
            )
        }

        false -> {
            UrlImage(
                url = profile,
                contentDescription = "profileImage",
                modifier = modifier,
            )
        }
    }
}
```

정도로 줄이고 isEmptyProfile은 빼도 되지 않을까? 상태가 이원화되어서 관리가 복잡한것 같아.

Reply:
처음에 아래처럼 했었는데, 모델에서 로직 적용 후 결과값을 넘겨주는게 맞나..해서 위처럼 구현했습니다.
저도 확신은 없었어서 누누씨 의견대로 가겠읍니다